src/data/automation_i18n.ts

```diff
@@ -349,7 +349,7 @@ const tryDescribeTrigger = (
   if (trigger.platform === "time" && trigger.at) {
     const result = ensureArray(trigger.at).map((at) =>
       typeof at !== "string"
-        ? at
+        ? `entity ${hass.states[at.entity_id] ? computeStateName(hass.states[at.entity_id]) : at.entity_id} ${at.offset ? " " + hass.localize(`${triggerTranslationBaseKey}.time.offset_by`, { offset: describeDuration(hass.locale, at.offset) }) : ""}`
```

Comment:
I'll follow the rabbit comment here when reviewing. The ternary operator is being used correctly and efficiently. However, combining multiple ternaries in a single line can make the code harder to read and maintain. It might be clearer to refactor this into if-else blocks for better readability.